api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -358,17 +358,37 @@ message SendLogsRequest {
   }
 }
 
-// Message to encapsulate setting the log level.
-message SetLogLevel {
-  // The log level to be set in the operator.
-  LogLevel level = 1;
-}
+//// Message to encapsulate setting the log level.
+//message SetLogLevel {
+//  // The log level to be set in the operator.
+//  LogLevel level = 1;
+//}
 
```

Comment:
Delete code instead of commenting it.
Trust GitHub to not lose the history of changes.
```suggestion
```